<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="dab_fav_gui">
     <widget class="QLabel" name="lbl_fm_fav_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;DAB favorites&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_to_tune">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>100</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TUNE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_to_fm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st03">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st06">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st04">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st01">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st02">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st05">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st07">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st09">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st10">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st11">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st12">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dab_st08">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fm_fav_gui">
     <widget class="QLabel" name="lbl_fm_fav">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;FM favorites&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st01">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st02">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st03">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st04">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_to_dab">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DAB</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_to_tune">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>100</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TUNE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st05">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st09">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st06">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st10">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st07">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st08">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>220</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st11">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fm_st12">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>330</y>
        <width>150</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="scan_list_gui">
     <widget class="QListWidget" name="lst_fm">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>260</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_rename_station">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>RENAME
STATION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVE
STATION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD
STATION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tune_to_fm">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>100</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tune_to_dab">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DAB</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_favorite">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD
FAVORITE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_rem_favorite">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVE
FAVORITE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_add_station">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>67</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Mhz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tuner_mode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>FM
MODE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_scan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>SCAN</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lst_dab">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>140</y>
        <width>260</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
